ocs_ci/ocs/resources/pod.py

```diff
@@ -2608,14 +2609,17 @@ def get_osd_pod_id(osd_pod):
 
 
 def get_pods_in_statuses(
-    status_options, namespace=config.ENV_DATA["cluster_namespace"]
+    status_options,
+    namespace=config.ENV_DATA["cluster_namespace"],
+    exclude_pod_name_prefixes=None,
 ):
     """
     Get all the pods in specific statuses
 
     Args:
         status_options (list): The list of the status options.
         namespace (str): Name of cluster namespace(default: config.ENV_DATA["cluster_namespace"])
+        exclude_pod_name_prefixes (list): The list of the pod name prefixes to exclude from the pods to get
```

Comment:
This line and few other lines in the docs strings are longer than 120 characters (in the [Coding Guidelines](https://ocs-ci.readthedocs.io/en/latest/coding_guidelines.html) we have even 79 characters, but [in tox.ini](https://github.com/red-hat-storage/ocs-ci/blob/master/tox.ini#L42) there is 120). I'm little bit confused how it passed the tox though.